core/src/main/java/io/questdb/griffin/CompiledQuery.java

```diff
@@ -42,12 +43,30 @@
     short COPY_REMOTE = 11;
     short RENAME_TABLE = 12;
     short BACKUP_TABLE = 13;
+    short LOCK = 14;
+    short UNLOCK = 14;
 
     RecordCursorFactory getRecordCursorFactory();
 
     InsertStatement getInsertStatement();
 
     TextLoader getTextLoader();
 
+    AlterStatement getAlterStatement();
+
     short getType();
+
+    /***
+     * Executes the query.
+     * If execution is done in sync returns an instance of QueryFuture where isDone() is true.
+     * If execution happened async, QueryFuture.await() method should be called for blocking wait
+     * or QueryFuture.await(long) for wait with specified timeout.
+     * @param tempSequence - temporary SCSequence instance to track completion of async ALTER command
+     * @throws SqlException - throws exception if command execution fails
+     */
+    QueryFuture execute(SCSequence tempSequence) throws SqlException;
```

Comment:
`subSeq` - important to state the role of the sequence in its name

Reply:
ok, made `eventSubSeq` as above